pyglet/input/darwin_hid.py

```diff
@@ -35,13 +35,19 @@
 from __future__ import print_function
 from __future__ import absolute_import
 from builtins import object
+
```

Comment:
Instead of using the line break ”\",  it's more readable to just have two lines of imports. For example: 
```from ctypes import cdll, util, CFUNCTYPE, c_int, c_ubyte, c_bool, c_void_p, 
from ctypes import c_uint32, c_uint32, c_uint64, byref
```